scripts/modules/elastic_stack.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, **options):
             ("apm-server.write_timeout", "1m"),
             ("logging.json", "true"),
             ("logging.metrics.enabled", "false"),
-            ("setup.kibana.host", self.DEFAULT_KIBANA_HOST),
```

Comment:
`setup.kibana.host` can actually be removed since we dropped 6.x support - that was used for loading dashboards into kibana.  You should set `apm-server.kibana.host` here instead:

https://github.com/elastic/apm-integration-testing/blob/33de17885a8bdc7fcc6b2d6f3b53ae318f403522/scripts/modules/elastic_stack.py#L78

Reply:
Also in the test? https://github.com/elastic/apm-integration-testing/pull/738/files#diff-13069991c5d90a2d74b80596f8686278R1407

Reply:
done